command/sub8_missions/missions/pose_sequence_test.py

```diff
@@ -0,0 +1,15 @@
+from txros import util
+import numpy as np
+from sub8.sub_singleton import PoseSequenceCommander
+
+
+@util.cancellableInlineCallbacks
+def run(sub_singleton):
+    print "Starting"
```

Comment:
Can you use a mil_misc_tools.text_effects printer? (Examples [here](https://github.com/uf-mil/SubjuGator/search?utf8=%E2%9C%93&q=fprint&type=) and [here](https://github.com/uf-mil/SubjuGator/search?utf8=%E2%9C%93&q=printer&type=))
This is the colorful printer we like to use for high level mission stuff and it's good for you to start getting used to it.